core/src/wallet/bitcoin/keychains/CommonBitcoinLikeKeychains.hpp

```diff
@@ -95,6 +95,8 @@ namespace ledger {
              */
             KeychainPersistentState getState() const;
 
+            std::string getKeychainEngine() const;
```

Comment:
In `core/src/bitcoin/BitcoinLikeAddress.hpp`, you return a `const std::string &` for the same method. Isn't it better to use the same signature here ?

Reply:
In addition, it's in best practices to return const & for getters whenever it's possible

Reply:
done